src/sentry/static/sentry/app/views/settings/organizationMembers/organizationMembersWrapper.tsx

```diff
@@ -126,33 +108,30 @@ class OrganizationMembersWrapper extends AsyncView<Props, State> {
           </Button>
         </StyledPanel>
 
-        {this.showNavTabs && (
-          <NavTabs underlined>
-            <ListLink
-              to={`/settings/${orgId}/members/`}
-              isActive={() => !this.onRequestsTab}
-              data-test-id="members-tab"
-            >
-              {t('Members')}
-            </ListLink>
-            <ListLink
-              to={`/settings/${orgId}/members/requests/`}
-              isActive={() => this.onRequestsTab}
-              data-test-id="requests-tab"
-              onClick={() => {
-                this.showInviteRequests &&
-                  trackAnalyticsEvent({
-                    eventKey: 'invite_request.tab_clicked',
-                    eventName: 'Invite Request Tab Clicked',
-                    organization_id: organization.id,
-                  });
-              }}
-            >
-              {t('Requests')}
-            </ListLink>
-            {this.requestCount && <StyledBadge text={this.requestCount} />}
-          </NavTabs>
-        )}
+        <NavTabs underlined>
+          <ListLink
+            to={`/settings/${orgId}/members/`}
+            isActive={() => !this.onRequestsTab}
+            data-test-id="members-tab"
+          >
+            {t('Members')}
+          </ListLink>
+          <ListLink
+            to={`/settings/${orgId}/members/requests/`}
+            isActive={() => this.onRequestsTab}
+            data-test-id="requests-tab"
+            onClick={() =>
+              trackAnalyticsEvent({
```

Comment:
Do we need to track every click? 

If we don't have to, can we have a private variable in `OrganizationMembersWrapper` and wrap it in `if (this.isFirstClick) {...}` so we are tracking 1 click per component mounted?